source/extensions/load_balancing_policies/override_host/override_host_filter_state.h

```diff
@@ -16,15 +17,22 @@ class OverrideHostFilterState : public StreamInfo::FilterState::Object {
 public:
   static constexpr absl::string_view kFilterStateKey =
       "envoy.extensions.load_balancing_policies.override_host.filter_state";
-  OverrideHostFilterState() = default;
 
-  uint64_t fallbackHostIndex() const { return fallback_host_index_; }
-  void setFallbackHostIndex(uint64_t fallback_host_index) {
-    fallback_host_index_ = fallback_host_index;
+  OverrideHostFilterState(std::string&& host_list) : host_list_(std::move(host_list)) {
+    for (absl::string_view host : absl::StrSplit(host_list_, ',', absl::SkipWhitespace())) {
```

Comment:
Please move this into a separate function, so it can be used in the class initializer list and make `selected_hosts_` const.

Reply:
SGTM.